composer.json

```diff
@@ -31,7 +31,7 @@
     ],
     "require" : {
         "php" : ">=7.1",
-        "qtism/qtism": "0.28.3"
+        "qtism/qtism": ">=0.28.3"
```

Comment:
Not sure about `>` here.
Let's keep it down to `~0` to avoid breaking changes.

Reply:
good point